src/app/app-routing.module.ts

```diff
@@ -1,8 +1,9 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-import { DemoPageComponent } from './pages/demo/demo.component';
+import { ArticlesComponent } from './components/articles/articles.component';
 
-const routes: Routes = [{ path: '', component: DemoPageComponent }];
+
+const routes: Routes = [{ path: '', component: ArticlesComponent }];
```

Comment:
Home component for home page makes ore sens

Reply:
Routes are coming in an other US. We will explain our choices when pushing that US